pages/index.vue

```diff
@@ -17,6 +17,15 @@
       :url="hero.url"
     />
     <b-container>
+      <header v-if="!hero">
```

Comment:
Could you limit the width of the header? On small screens this if fine, but on larger screens it can be limited to 75% for readability.